deepchecks/core/serialization/common.py

```diff
@@ -159,11 +160,11 @@ def aggregate_conditions(
 
     df = pd.DataFrame(
         data=data,
-        columns=['Status', 'Check', 'Condition', 'More Info', 'sort']
+        columns=['Status', 'Check', 'Condition', 'More Info', '$priority']
     )
 
-    df.sort_values(by=['sort'], inplace=True)
-    df.drop('sort', axis=1, inplace=True)
+    df.sort_values(by=['$priority'], inplace=True)
```

Comment:
why do you need the sort here?

Reply:
we sort condition results here, not check results